CAIPs/caip-24.md

```diff
@@ -0,0 +1,70 @@
+---
+caip: 24
+title: Asset Reference for the cosmos Asset Namespace
+author: Theobold Frazier
+discussions-to: https://github.com/shapeshift/CAIPs/issues/1
+status: Draft
+type: Standard
+created: 2020-02-14
+updated: 2020-02-14
+requires: 19
+---
+
+## Simple Summary
+
+This document is about the details of the Cosmos network asset namespace and reference for CAIP-19.
+
+## Abstract
+
+In CAIP-19 a general asset identification scheme is defined. This is the
+implementation of CAIP-19 for Cosmos chain native assets and ibc assets.
+
+## Motivation
+
+See CAIP-19.
+
+## Specification
+
+### Cosmos chain Native Asset Namespace
+
+An asset namespace called "native" that references cosmos network blockchain native
+assets, allowing for unique identification on cosmos chains with multiple native assets.
+
+### Cosmos chain ibc asset Asset Namespace
+
+The asset namespace is called "ibc". It reference cosmos network blockchain inter blockchain channel assets.
```

Comment:
It would be good to specify what data is used to determine the asset id. I know we're using `base` but I'm not sure if that's something Cosmos specific or just something that Osmosis chose to use.

Reply:
Added a section for IBC Asset Reference definition. base denomination seem to be more generally applied to the cosmos ecosystem.
https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-024-coin-metadata.md